.circleci/config.yml

```diff
@@ -1453,4 +1453,5 @@ workflows:
               only:
                 - master
     jobs:
-      - docker_system_tests
+      - docker_system_tests:
```

Comment:
This looks the only scheduled workflow. Will this work? Maybe to be moved to other org?

Reply:
🤔